business_requirement/models/business.py

```diff
@@ -0,0 +1,481 @@
+# -*- coding: utf-8 -*-
+# © 2016 Elico Corp (https://www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, fields, models, _
+from openerp.exceptions import except_orm
+from openerp import tools
+from openerp import SUPERUSER_ID
+
+
+class BusinessRequirement(models.Model):
+    _inherit = ['mail.thread', 'ir.needaction_mixin']
+    _name = "business.requirement"
+    _description = "Business Requirement"
+    _order = 'id desc'
+
+    @api.model
+    def _get_default_company(self):
+        company_id = self.env.user._get_company()
+        if not company_id:
+            raise except_orm(
+                _('Error!'),
+                _('There is no default company for the current user!'))
+        return self.env['res.company'].browse(company_id)
+
+    sequence = fields.Char(
+        'Sequence',
+        readonly=True,
+        copy=False,
+        index=True,
+    )
+    name = fields.Char(
+        'Name',
+        required=False,
+        readonly=True,
+        copy=False,
+        states={'draft': [('readonly', False)]}
+    )
+    description = fields.Char(
+        'Description', required=True,
+        readonly=True,
+        states={'draft': [('readonly', False)]}
+    )
+    business_requirement = fields.Text(
+        'Customer Story',
+        readonly=True,
+        states={'draft': [('readonly', False)]}
+    )
+    scenario = fields.Text(
+        'Scenario',
+        readonly=True,
+        states={'draft': [('readonly', False)]}
+    )
+    gap = fields.Text(
+        'Gap',
+        readonly=True,
+        states={'draft': [('readonly', False)]}
+    )
+    categ_id = fields.Many2many(
+        'project.category',
```

Comment:
@elicoidal depends if you want to have the same tags (project.category) for `project.task` and `business.requirement` or have separated ones.
Other comment, yes the tags should be searchable in tree view.